src/tasks/displayAnalysisResult.js

```diff
@@ -33,20 +37,21 @@ const displayAnalysisResults = (result, isFree, isDistant) => {
             console.info(`✅ ${scenario.name} completed`);
             if (scenario.threshold) {
                 console.info(
-                    `The estimated footprint at ${totalCo2} g eq. co2 ± ${precision}% (${totalMWh} Wh) is under the limit configured at ${scenario.threshold} g eq. co2.`
+                    `The estimated footprint at ${totalCo2} eq. co2 ± ${precision}% (${totalMWh}) is under the limit configured at ${scenario.threshold} g eq. co2.`
                 );
             } else {
                 console.info(
-                    `The estimated footprint is ${totalCo2} g eq. co2 ± ${precision}% (${totalMWh} Wh).`
+                    `The estimated footprint is ${totalCo2} eq. co2 ± ${precision}% (${totalMWh}).`
                 );
             }
 
             if (scenario.executionCount) {
                 console.info(
                     `For ${
                         scenario.executionCount
-                    } scenario executions, this represents ${formatTotalCo2(
-                        totalCo2 * scenario.executionCount
+                    } scenario executions, this represents ${computeTotalMetric(
+                        scenario.score.co2.total * scenario.executionCount,
```

Comment:
I think you should securise variable with "?."
Also, there is only one argument in computeTotalMetric,  no unit 
Something like 
`this represents ${computeTotalMetric(scenario?.score?.co2?.total * scenario.executionCount)} g`